README.md

```diff
@@ -1 +1,45 @@
-# calculator
\ No newline at end of file
+## Calculator App
+"Calculator App" is a signle page application that do basic calculations.
+
+## Built With
+- JavaScript
+- HTML
+- CSS
+- Webpack
+- React
+
+## Getting Started
+To get a local copy up and running follow these simple example steps on the command line.
+
+## Clone the project from it's github repo.
+
+> ### $ git clone https:/githubcom/denisdiaconumath-magicians.git 
+  
+## Change directory to the root of project
+
+> ### $ cd calculator
+  
+## Install all dev dependencies
+
+> ###  $ npm install
+
+
+## Run the server
+
+> ### $ npm start 
+
+## Author
+
+👤 **Fares Ashraf** 
+
+[![](https://img.shields.io/badge/GitHub-100000?style=for-the-badge&logo=github&logoColor=white)](https://github.com/ashraffares) [![](https://img.shields.io/badge/Twitter-1DA1F2?style=for-the-badge&logo=twitter&logoColor=white)](https://twitter.com/Fares09301164) [![](https://img.shields.io/badge/LinkedIn-0077B5?style=for-the-badge&logo=linkedin&logoColor=white)](https://www.linkedin.com/in/faresashraf/)
+
+## 🤝 Contributing
+
+Contributions, issues and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/ashraffares/Phaser_Game/issues).
```

Comment:
- [ ] [REQUIRED] Please make sure that this URL (issues pages) redirects to the correct repo.